### ISSUE TITLE
Bump to Ansible-2.7

### ISSUE THREAD

**Issue by ansell** [AtlasOfLivingAustralia/ala-install#202]
Ansible-2.7 was released 3 weeks ago. Tasks for bumping to it are:

- [x] Bump supported version number in ``ansible/group_vars/all/vars.yml`` https://github.com/AtlasOfLivingAustralia/ala-install/commit/cec7a8b5622f9764ac7f34edfe8ddf7f0368df2b
- [x] Test a variety of playbooks locally
- [x] Upgrade on ci jenkins 
- [x] Run all ci jenkins jobs to verify that all of the playbooks that are not previously failing still work
- [x] Investigate failure in mysql role with error message ``'update_cache' is not a valid attribute for a Task The error appears to have been in '<https://ci.ala.org.au/job/Ansible%20-%20Logger%20Service/ws/ansible/roles/mysql/tasks/main.yml'>: line 10, column 3`` https://github.com/AtlasOfLivingAustralia/ala-install/commit/1d53ef2222282f6932c6ce41885efc2fcbfd45e3
- [x] Investigate failure in tomcat vhost/bie-hub ```TASK [bie-hub : create tomcat vhost] An exception occurred during task execution. To see the full traceback, use -vvv. The error was: SyntaxError: Missing parentheses in call to 'print'. Did you mean print(remote_internal_proxies)?``` The bug here is a python-2 to python-3 issue, you can’t call print without brackets in python-3. Either ansible or brew seem to have switched to using python-3 and it broke our custom python code: https://github.com/AtlasOfLivingAustralia/ala-install/blob/master/ansible/library/tomcat_vhost#L198
- [ ] Fix deprecation warning relating to our common use of apt with a loop. ```TASK [java : Install Oracle Java 8 (before possibly patching files)] [DEPRECATION WARNING]: Invoking "apt" only once while using a loop via squash_actions is deprecated. Instead of using a loop to supply multiple items and specifying `name: {{item}}`, please use `name: ['oracle-
java8-installer', 'ca-certificates', 'oracle-java8-set-default']` and remove the loop. This feature will be removed in version 2.11. Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.```
- [x] Fix use of python-2 syntax for exception handling ```except Exception, e``` is used in a few places in our custom code in ``ansible/library``
- [ ] Update advertised supported version in ``README.md`` file

**Comment by ansell** [comment]
Opened a new issue to track the apt items deprecation warning as it isn't blocking the use of 2.7.0: https://github.com/AtlasOfLivingAustralia/ala-install/issues/206
